src/main/java/com/farao_community/farao/gridcapa/job_launcher/JobLauncherAutoService.java

```diff
@@ -50,26 +46,35 @@ public Consumer<Flux<TaskDto>> consumeTaskDtoUpdate() {
                 .subscribe(this::runReadyTasks);
     }
 
-    void runReadyTasks(TaskDto taskDtoUpdated) {
+    void runReadyTasks(TaskDto updatedTaskDto) {
         try {
-            if (taskDtoUpdated.getStatus().equals(TaskStatus.READY)) {
-                // propagate in logs MDC the task id as an extra field to be able to match microservices logs with calculation tasks.
+            if (updatedTaskDto.getStatus().equals(TaskStatus.READY)) {
```

Comment:
Reverse order of comparisons throughout the proposed changes so that it match the standard : REFERENCE_VALUE.equals(value_to_check) to avoid NPE